src/applications/disability-benefits/all-claims/tests/utils/utils.unit.spec.jsx

```diff
@@ -1,57 +1,58 @@
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
 import moment from 'moment';
-import { minYear, maxYear } from 'platform/forms-system/src/js/helpers';
+import { maxYear, minYear } from 'platform/forms-system/src/js/helpers';
 import { checkboxGroupSchema } from 'platform/forms-system/src/js/web-component-patterns';
 
 import {
-  SAVED_SEPARATION_DATE,
-  PTSD_MATCHES,
   CHAR_LIMITS,
+  PTSD_MATCHES,
+  SAVED_SEPARATION_DATE,
 } from '../../constants';
 import {
+  activeServicePeriods,
+  baseDoNew4142Logic,
   capitalizeEachWord,
   fieldsHaveInput,
+  formatDate,
+  formatDateRange,
+  formatFullName,
+  formatMonthYearDate,
   hasGuardOrReservePeriod,
   hasHospitalCare,
+  hasNewPtsdDisability,
   hasOtherEvidence,
   increaseOnly,
   isAnswering781aQuestions,
   isAnswering781Questions,
+  isBDD,
+  isDisabilityPtsd,
+  isExpired,
+  isNotExpired,
+  isUndefined,
   isUploading781aForm,
   isUploading781aSupportingDocuments,
   isUploading781Form,
+  isValidFullDate,
+  isValidServicePeriod,
   isWithinRange,
+  makeConditionsSchema,
```

Comment:
Boy this linting behavior is really adding a lot of lines to PRs and making it difficult to know what's new and what's not haha, did we ever figure out what is doing this and if we can shut it off?

Reply:
Its my editor setting somewhere I had turned it off and deleted and recloned the repo need to add the setting back.

Although it is really nice to have imports sorted.